### PR TITLE
Adds Docker Support and uses RC for Config

### PR DESCRIPTION
**Pull request by ekristen**

### CODE REVIEW

**Comment by yannickglt** [comment]
+1

File: package.json [hunk 1]
```diff
@@ -50,6 +51,7 @@
     "moment": "^2.9.0",
     "npm": "^3.3.12",
     "nsp-advisories-api": "^2.0.2",
+    "rc": "^1.1.5",
```
Comment (alanshaw) [inline package.json]: Could you remove the dependency on `config` now we're using `rc`?

File: package.json [hunk 2]
```diff
@@ -61,6 +63,9 @@
     "brfs": "^1.4.0",
     "grunt": "^0.4.5",
     "grunt-browserify": "^4.0.0",
+    "grunt-cli": "^0.1.13",
+    "grunt-contrib-clean": "^0.7.0",
+    "grunt-cli": "^0.1.13",
```
Comment (alanshaw) [inline package.json]: Two `grunt-cli` here...

File: .gitignore [hunk 3]
```diff
@@ -4,3 +4,4 @@ data/
 dist/
 node_modules/
 npm-debug.log
+.*rc
```
Comment (alanshaw) [inline .gitignore]: Could you also remove the `config/runtime.json` from here?

**Comment by alanshaw** [comment]
Looks, good. I've just commented on a couple of minor points.

**Comment by ekristen** [comment]
Odd, I thought I had taken care of the multiple deps problem. All fixes requested are in.

**Comment by alanshaw** [comment]
:thumbsup: thank you

**Pull request merged**